scripts/monitor.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+source .env
+
+log() {
+    echo "$(date +'%Y-%m-%d %H:%M:%S') - $1"
+}
+
+send_alert() {
+    message=$1
+    logs=$2
+    echo "Sending alert to slack $CHANNEL ... $message"
+
+    curl -X POST "https://slack.com/api/chat.postMessage" \
+        -H "Authorization: Bearer $BOT_AUTH_TOKEN" \
+        -H 'Content-type: application/json; charset=utf-8' \
+        --data '{
+            "channel": "'"$CHANNEL"'",
+            "text": "'"$message"'\n'"$logs"'"
+        }' > /dev/null 2>&1
+}
+
+get_container_names() {
```

Comment:
Public prover nodes may run other docke containers so in fact we may want hard code the default container names and check them. Currently most prover provider will use default name. If they do not use default name it is their responsibility to change this scripts of the name.